src/components/icon-as-button/IconAsButton.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import Icon, {types, colors as iconColors} from '../icons/Icon';
+
+const colors = {
```

Comment:
icon-as-button have less amount of colors than normal icons. And it has not perfectly  written scss. I created issue on github https://github.com/brainly/style-guide/issues/1018